app/views/search/index.html.erb

```diff
@@ -95,7 +95,7 @@
                   <span class="address__line post-code"><%= firm.address_postcode %></span>
                 </p>
 
-                <a href="tel:<%= firm.telephone_number.strip %>" class="address__link phone">
+                <a href="tel:<%= firm.telephone_number.gsub(/\s+/, '') %>" class="address__link phone">
```

Comment:
I was thinking about this. We need to do it in both places and to be honest it's probably worth biting the bullet and pushing this into the actual `FirmResult` over in core.